modules/monitoring-creating-cluster-id-labels-for-metrics.adoc

```diff
@@ -0,0 +1,100 @@
+** You have created the `user-workload-monitoring-config` `ConfigMap` object.
+* You have installed the OpenShift CLI (`oc`).
+* You have configured remote write storage.
+
+.Procedure
+
+Follow these steps to configure cluster ID labels for metrics for default platform monitoring in the `cluster-monitoring-config` config map in the `openshift-monitoring` namespace.
+
+[NOTE]
+====
+If you configure cluster ID labels for metrics for the Prometheus instance that monitors user-defined projects, make similar edits to the `user-workload-monitoring-config` config map in the `openshift-user-workload-monitoring` namespace. 
+Note that the Prometheus config map component is called `prometheus` in the `user-workload-monitoring-config` `ConfigMap` object and not `prometheusK8s`, as it is in the `cluster-monitoring-config` `ConfigMap` object. 
+====
+
+. Edit the `cluster-monitoring-config` `ConfigMap` object in the `openshift-monitoring` project:
++
+[source,terminal]
+----
+$ oc -n openshift-monitoring edit configmap cluster-monitoring-config
+----
+
+. In the `writeRelabelConfigs:` section under `data/config.yaml/prometheusK8s/remoteWrite`, add cluster ID label configuration values:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    prometheusK8s:
+      remoteWrite:
+      - url: "https://remote-write-endpoint.example.com"
+        <endpoint_authentication_credentials>
+        <write_relabel_configs> <1>
+          - <label_config> <2>
+----
+<1> The write relabel configuration settings.
+<2> The source and target label configuration settings for the metrics sent to remote write storage.
++
+For `<write_relabel_configs>`, substitute a list of write relabel configurations for metrics that you want to send to the remote endpoint.
++
+For `<label_config>`, substitute the label configuration for the metrics sent to the remote endpoint.
++
+The following sample shows how to forward a single metric called `my_metric` with the cluster ID label `cluster_id_1`:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    prometheusK8s:
+      remoteWrite:
+      - url: "https://remote-write-endpoint.example.com"
+        writeRelabelConfigs:
+        - sourceLabels: [__name__]
+          regex: 'my_metric'
+          action: keep
+        - sourceLabels:
+            - __tmp_openshift_cluster_id__ <1>
+          targetLabel: cluster_id_1 <2>
+          action: replace <3>
+----
+<1> The name of a temporary source label for the cluster ID label. This label must begin with [x-]`__tmp_openshift_` followed by the cluster ID label you want to add followed by [x-]`__` as indicated in the sample.
```

Comment:
Small correction: The source label is always going to be `__tmp_openshift_cluster_id__` for this cluster id feature. More details for context, this doesn't need to be part of the docs:
CMO adds this temporary label by adding two relabel steps to the `writeRelabelConfigs` of all `remoteWrite` sections. It adds an entry as the first step to add the label called `__tmp_openshift_cluster_id__` with the value of the cluster id. It also adds a step to the end of the `writeRelabelConfigs` removing the label called `__tmp_openshift_cluster_id__`.
A user can add a relabel step as documented here to take the value of this temporary label and add it to their own (non-temporary label). They can choose the name of that label as they see fit.

Reply:
Revised per your explanation.